visual-inspection/inspection-backend/inspection/explainer/explainers/lime_.py

```diff
@@ -37,27 +38,47 @@ class Config:
 @traced(label="compute_explanation", attributes={"explanation.method": "lime"})
 def lime_explanation(input_img: np.ndarray,
                      model_: tf.keras.models.Model,
+                     index_of_label_to_explain: int,
+                     positive_only_parameter: bool,
                      **settings) -> np.ndarray:
     config = LIMEConfiguration(**settings)
 
     explanation = _lime.explain_instance(input_img.astype("double"), model_.predict,
                                          top_labels=config.explainer.top_labels,
                                          num_features=config.explainer.num_features,
                                          num_samples=config.explainer.num_samples)
-
-    return render_explanation(explanation, config.renderer)
+    #return render_explanation(explanation, index_of_label_to_explain, positive_only_parameter, config.renderer)
+    return render_explanationV2(input_img, explanation, index_of_label_to_explain, positive_only_parameter, config.renderer)
 
 
 @traced
-def render_explanation(explanation: lime_image.ImageExplanation,
+def render_explanation(explanation: lime_image.ImageExplanation, index_of_label_to_explain : int, positive_only_parameter : bool,
                        config: RendererConfiguration):
     image, mask = explanation.get_image_and_mask(
-        explanation.top_labels[0],
-        positive_only=config.positive_only,
+        explanation.top_labels[index_of_label_to_explain],
+        positive_only= positive_only_parameter,#config.positive_only,
         negative_only=False,
         num_features=config.num_features,
         min_weight=config.min_weight,
-        hide_rest=False
+        hide_rest= True #config.hide_rest
     )
 
     return mark_boundaries(image / 2 + 0.5, mask)
+@traced
+def render_explanationV2(input_img: np.ndarray, explanation: lime_image.ImageExplanation, index_of_label_to_explain : int,                                            positive_only_parameter: bool, config: RendererConfiguration):
```

Comment:
Please do not introduce multiple versions of functions, especially if the original version cannot be reached anymore anyway. If you need a changed version of a function, just change it 😃